demos/wait-for-pods-ready/resources/service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sample-server
+  namespace: demo
+  labels:
+    app: sample-app
+spec:
+  containers:
+    - name: server
+      image: quay.io/rh_ee_czaccari/python:3.9-slim
+      command: ["python", "-m", "http.server", "80"]
+      ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sample-service
+  namespace: demo
+spec:
+  selector:
+    app: sample-app
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
```

Comment:
Quick q, is the service necessary for this example?

Reply:
It is necessary to expose the 'server' to other pods. In this example, the Jobs will attempt curl the host, which can only be accessed through the Service.

Reply:
ack